packages/template-retail-react-app/app/hooks/use-einstein.js

```diff
@@ -404,7 +409,8 @@ const useEinstein = () => {
                 host,
                 einsteinId,
                 siteId,
-                isProduction
+                isProduction,
+                finalDntValue
```

Comment:
The EinsteinAPI is taking `dnt` as a prop in constructor not `finalDntValue`
```suggestion
                dnt: finalDntValue
```